packages/nextjs/hooks/scaffold-eth/useScaffoldContractWrite.ts

```diff
@@ -18,7 +19,7 @@ export const useScaffoldContractWrite = (contractName: string, functionName: str
     args,
     functionName,
     overrides: {
-      value: value ? parseTxnValue(value) : undefined,
+      value: value ? utils.parseEther("1").mul(parseTxnValue(value)) : undefined,
```

Comment:
![image](https://user-images.githubusercontent.com/2486142/216563800-da1076d3-a79e-4856-9c6f-2462211f0047.png)

This was giving me an error when trying a decimal value.

I think this should be enough c9897b02a56cda27ea7fbb7c9f026956102ff08c. Maybe I'm missing an use case where it breaks... but we can fix later if we encounter an issue.